js/src/search/SearchVisualizer.ts

```diff
@@ -1,15 +1,15 @@
 import * as widgets from "@jupyter-widgets/base";
 import { timeout } from "d3";
+import * as _ from "lodash";
```

Comment:
seems like a better idea to just import the one function that you will be using, instead of everything

Reply:
fixed